primary PR - readme and configuration files are created (#2)
nemgrouplimited/symbol-node-monitoring-agent: README.md

> # Symbol Node Monitoring Agent
> Symbol node monitoring agent is a small rest application that runs inside a catapult server box exercising the node testing. It is mandatory to have this application running either as part of symbol-bootstrap or manually colocated with the Symbol Node in order to be part of Symbol Node Rewards Programme.
> 
> ## Installation & Running
> 
> Please visit the [documentation page](https://docs.symbolplatform.com/guides/network/supernode-enrollment.html#manually) for the instructions.
> 
> *Please note that you can use the following command to generate the agent certificates.

Reviewer: ```suggestion
# Symbol Node Monitoring Agent
The Symbol node monitoring agent is a small REST application that runs inside a catapult server machine performing the node testing. It is mandatory to have this application running (deployed by either symbol-bootstrap or manually) in order to be part of the Symbol Node Rewards Programs.

## Installation & Running

Please visit the [documentation page](https://docs.symbolplatform.com/guides/network/supernode-enrollment.html#manually) for the instructions.

You can use the following command to generate the agent certificates:
```

```
bash ./cmds/init.sh
```

> This will create the certificates in the 'certs' directory.

Reviewer: ```suggestion
This will create the certificates in the `certs` directory.
```
